### PR TITLE
XW-1725 | add tracking to all links in footer

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1725

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/initializers/global-footer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			track({
+				action: trackActions.click,
+				category: 'footer',
+				label: trackingLabel
+			});
+		}
+
+		if (event.target.getAttribute('id') === 'global-footer-full-site-link') {
+			const domainNameRegExpMatchArray = (/\.[a-z0-9\-]+\.[a-z0-9]{2,}$/i).exec(window.location.hostname),
+				cookieDomain = domainNameRegExpMatchArray ? domainNameRegExpMatchArray[0] : '',
```
Comment (rogatty) [inline front/main/app/initializers/global-footer.js]: Can you address https://github.com/Wikia/mercury/pull/2706#discussion_r74371003 here so we won't have unnecessary conflicts?

File: front/main/app/initializers/global-footer.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import {track, trackActions} from 'common/utils/track';
+
+const {$, getWithDefault} = Ember;
+
+/**
+ * @returns {void}
+ */
+export function initialize() {
+	$('.wds-global-footer a').click((event) => {
+		const trackingLabel = event.currentTarget.getAttribute('data-tracking-label');
```
Comment (rogatty) [inline front/main/app/initializers/global-footer.js]: You can also use `this`, see https://api.jquery.com/event.currentTarget/

File: front/main/app/initializers/global-footer.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+		if (trackingLabel) {
+			track({
+				action: trackActions.click,
+				category: 'footer',
+				label: trackingLabel
+			});
+		}
+
+		if (event.target.getAttribute('id') === 'global-footer-full-site-link') {
```
Comment (rogatty) [inline front/main/app/initializers/global-footer.js]: Shouldn't this be `event.currentTarget` / `this`? Same for `window.location.assign(event.target.href);`

File: server/app/views/_helpers/wrap-me.js [hunk 4]
```diff
@@ -40,5 +41,9 @@ module.exports = function (param, options) {
 		className = ` class="${escapeHtml(options.className)}"`;
```
Comment (rogatty) [inline server/app/views/_helpers/wrap-me.js]: As you're here can you fix it? Should be `hash.className`

File: front/main/app/initializers/global-footer.js [hunk 5]
```diff
@@ -29,7 +29,7 @@ export function initialize() {
 				path: '/'
 			});
 
-			window.location.assign(event.target.href);
+			window.location.assign(this.href);
```
Comment (rogatty) [inline front/main/app/initializers/global-footer.js]: `this.getAttribute('href')`?

**Comment by rogatty** [comment]
One thing and will LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**